docs/static/child.conf

```diff
@@ -17,7 +17,7 @@ BotHost "geeknode.org"
 
 # Host and port to connect to
 RemoteServer localhost
-Port 6999
+Port 7001
```

Comment:
What's wrong with 6999?

Reply:
Nothing, I used the docker files as an example. I don't remember why I had picked 7001 instead of 6999 though.